CHANGELOG

```diff
@@ -1,6 +1,7 @@
 # 3.0.5 (2020-XX-XX)
 
  * n/a
+ * Allow trailing commas in argument lists (in calls as well as definitions)
```

Comment:
Can you do it in 2.x instead?

Reply:
We might even add support in 1.x as the patch will probably apply cleanly.

Reply:
Sure thing, I will reapply it on 1.x

Reply:
Done.

Reply:
> as the patch will probably apply cleanly.

It was exactly the same code change. The only change I needed to do was to import the macro specifically in the macro test, as `_self` is deprecated in v1.